Gems/Camera/Code/Source/ViewportCameraSelectorWindow.cpp

```diff
@@ -207,12 +226,22 @@ namespace Camera
         {
             if (current.row() != previous.row())
             {
-                bool lockCameraMovement = false;
-
                 QScopedValueRollback<bool> rb(m_ignoreViewportViewEntityChanged, true);
-                AZ::EntityId entityId = selectionModel()->currentIndex().data(Qt::CameraIdRole).value<AZ::EntityId>();
-                EditorCameraRequests::Bus::Broadcast(
-                    &EditorCameraRequests::SetViewAndMovementLockFromEntityPerspective, entityId, lockCameraMovement);
+
+                const AZ::EntityId entityId = selectionModel()->currentIndex().data(Qt::CameraIdRole).value<AZ::EntityId>();
+                // only check entity validity if entity is valid, otherwise the change event will be for the editor camera
+                if (entityId.IsValid())
+                {
+                    // if the entity is not in an active state we are most likely in a transition event (going to game mode
+                    // or changing level) and we do not want so signal any changes to the camera request bus
+                    const AZ::Entity* entity = AzToolsFramework::GetEntityById(entityId);
+                    if (!entity || (entity && entity->GetState() != AZ::Entity::State::Active))
```

Comment:
nit: The second `entity` check is redundant, the second part of the || would only be done if entity is valid.

````
if (!entity || entity->GetState() != AZ::Entity::State::Active)
````

Reply:
Ah good spot! Will update :+1:

Reply:
Test re-added and description updated with output from missed test, thanks!